backend/tests/api/test_main.py

```diff
@@ -66,9 +69,8 @@ def test_remove_food(self):
             "decription": "victory hw",
             "category": "Vegetables",
             "serving_size": "1",
-            "food_collect_id": 1,
         }
-        response = self.app.post("/add_food", json=add_food_payload)
+        response = self.app.post("/add_food/1", json=add_food_payload)
```

Comment:
You did an awesome job making the route more flexible above using the route variable. Could you also do this for the remove food for consistency? It will just be easier in the future to change 1 variable instead of changing every test with an /add_food/1